plugins/woocommerce/src/Admin/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php

```diff
@@ -0,0 +1,773 @@
+			[
+				'id'         => 'product-inventory-advanced',
+				'blockName'  => 'woocommerce/product-collapsible',
+				'attributes' => [
+					'toggleText'       => __( 'Advanced', 'woocommerce' ),
+					'initialCollapsed' => true,
+					'persistRender'    => true,
+				],
+			]
+		);
+		$product_inventory_advanced_wrapper = $product_inventory_advanced->add_block(
+			[
+				'blockName'  => 'woocommerce/product-section',
+				'attributes' => [
+					'blockGap' => 'unit-40',
+				],
+			]
+		);
+		$product_out_of_stock_conditional   = $product_inventory_advanced_wrapper->add_block(
+			[
+				'id'         => 'product-out-of-stock-conditional-wrapper',
+				'blockName'  => 'woocommerce/conditional',
+				'attributes' => [
+					'mustMatch' => [
+						'manage_stock' => [ true ],
+					],
+				],
+			]
+		);
+		$product_out_of_stock_conditional->add_block(
+			[
+				'id'         => 'product-out-of-stock',
+				'blockName'  => 'woocommerce/product-radio-field',
+				'attributes' => [
+					'title'    => __( 'When out of stock', 'woocommerce' ),
+					'property' => 'backorders',
+					'options'  => [
+						[
+							'label' => __( 'Allow purchases', 'woocommerce' ),
+							'value' => 'yes',
+						],
+						[
+							'label' => __(
+								'Allow purchases, but notify customers',
+								'woocommerce'
+							),
+							'value' => 'notify',
+						],
+						[
+							'label' => __( "Don't allow purchases", 'woocommerce' ),
+							'value' => 'no',
+						],
+					],
+				],
+			]
+		);
+		$product_out_of_stock_conditional->add_block(
+			[
+				'id'        => 'product-inventory-email',
+				'blockName' => 'woocommerce/product-inventory-email-field',
+			]
+		);
+
+		$product_inventory_advanced_wrapper->add_block(
+			[
+				'id'         => 'product-limit-purchase',
+				'blockName'  => 'woocommerce/product-checkbox-field',
+				'attributes' => [
+					'title'    => __(
+						'Restrictions',
+						'woocommerce'
+					),
+					'label'    => __(
+						'Limit purchases to 1 item per order',
+						'woocommerce'
+					),
+					'property' => 'sold_individually',
+					'tooltip'  => __(
+						'When checked, customers will be able to purchase only 1 item in a single order. This is particularly useful for items that have limited quantity, like art or handmade goods.',
+						'woocommerce'
+					),
+				],
+			]
+		);
+	}
+
+	/**
+	 * Adds the shipping group blocks to the template.
+	 */
+	private function add_shipping_group_blocks() {
+		$shipping_group = $this->add_group(
+			[
+				'id'         => 'shipping',
+				'order'      => 40,
+				'attributes' => [
+					'id'    => 'shipping',
+					'title' => __( 'Shipping', 'woocommerce' ),
+				],
+			]
+		);
+		$shipping_group->add_block(
+			[
+				'id'         => 'product_variation_notice_shipping_tab',
+				'blockName'  => 'woocommerce/product-has-variations-notice',
+				'attributes' => [
+					'id'         => 'wc-product-notice-has-options',
```

Comment:
Another double `id` situation.

And, the name of the block `id` seems to indicate it is a tab.

Reply:
Removed now.